lib/diagnostic.js

```diff
@@ -5,16 +5,21 @@
 // the minimum value in an array
 // this callback will be passed by the tests to the reduce method, e.g.
 // `let minimum = array.reduce(min);`
-const min = function (/* previous, current */) {
+const min = function (array)
+{
 
+  let minimum = array.reduce((p, e, i, a) => p+c);
```

Comment:
I think you're not quite understanding the question here. `min` is supposed to be a function that takes previous and current and is *passed* to reduce as a callback. You do not need to use reduce to answer this question.
```js
const min = function (previous, current) {
  if (previous < current) {
    return previous;
  } 
  return current;
}
```